pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -93,17 +94,30 @@ RUN cd /tmp \
   && make -j64 src.build BUILDDIR=/usr/local \
   && rm -rf /tmp/nccl
 
-# Install EFA along with AWS OPEN_MPI
+# Install EFA along without AWS OPEN_MPI
```

Comment:
With this, will we still require `RDMAV_FORK_SAFE=1` to be set on the dockerfile's env variables?

Reply:
Same reply as above